csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -93,78 +93,6 @@
-          PackagePath="\runtimes\win-$(TargetArchitecture)\native"
-          Pack="false"
-          CopyToOutputDirectory="Always"
-          Visible="false"
-    />
-    <None Include="$(NativeBuildOutputDir)\Microsoft.AI.MachineLearning.lib"
-          Condition="Exists('$(NativeBuildOutputDir)\Microsoft.AI.MachineLearning.lib')
-                     And ('$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.DirectML'
-                     Or '$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime')"
-          PackagePath="\runtimes\win-$(TargetArchitecture)\native"
-          Pack="false"
-          CopyToOutputDirectory="Never"
-          Visible="false"
-    />
-    <None Include="$(NativeBuildOutputDir)\Microsoft.AI.MachineLearning.pdb"
-          Condition="Exists('$(NativeBuildOutputDir)\Microsoft.AI.MachineLearning.pdb')
-                     And ('$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.DirectML'
-                     Or '$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime')"
-          PackagePath="\runtimes\win-$(TargetArchitecture)\native"
-          Pack="false"
-          CopyToOutputDirectory="Always"
-          Visible="false"
-    />
-    <None Include="$(OnnxRuntimeBuildDirectory)\$(Configuration)\Microsoft.AI.MachineLearning.winmd"
-          Condition="Exists('$(OnnxRuntimeBuildDirectory)\$(Configuration)\Microsoft.AI.MachineLearning.winmd')
-                     And ('$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.DirectML'
-                     Or '$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime')"
-          PackagePath="\build\native\metadata"
-          Pack="false"
-          CopyToOutputDirectory="Always"
-          Visible="false"
-    />
-    <None Include="$(OnnxRuntimeBuildDirectory)\$(Configuration)\Microsoft.AI.MachineLearning.h"
-          Condition="Exists('$(OnnxRuntimeBuildDirectory)\$(Configuration)\Microsoft.AI.MachineLearning.h')
-                     And ('$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.DirectML'
-                     Or '$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime')"
-          PackagePath="\build\native\include"
-          Pack="false"
-          CopyToOutputDirectory="Never"
-          Visible="false"
-    />
-    <None Include="$(OnnxRuntimeBuildDirectory)\$(Configuration)\Microsoft.AI.MachineLearning.Native.h"
-          Condition="Exists('$(OnnxRuntimeBuildDirectory)\$(Configuration)\Microsoft.AI.MachineLearning.Native.h')
-                     And ('$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.DirectML'
-                     Or '$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime')"
-          PackagePath="\build\native\include"
-          Pack="false"
-          CopyToOutputDirectory="Never"
-          Visible="false"
-    />
-    <None Include="$(OnnxRuntimeRoot)\winml\api\dualapipartitionattribute.h"
-          Condition="Exists('$(OnnxRuntimeRoot)\winml\api\dualapipartitionattribute.h')
-                     And ('$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.DirectML'
```

Comment:
>OnnxRuntime [](start = 61, length = 11)

Where does the DirectML pacakge get built now?

Reply:
It gets built in the Nuget GPU Pipeline Job for DML (x64/x86) and the Nuget WindowsAI Pipeline (x64/x86)

Packaging hasn't changed for the DirectML nuget package - and it gets binplaced in that package and the new Microsoft.AI.MachineLearning nuget package